assets/messages.yml

```diff
@@ -180,6 +224,24 @@ messages:
         The authentication services are offline.
         This will be fixed shortly, please hang tight. Check tweets from [@MojangSupport|https://twitter.com/MojangSupport] for the current status.
 
+        %quick_links%
+      fillname: []
+  awaiting-response:
+    - project:
+        - bds
+        - mc
+        - mcl
+        - mcd
+        - mce
+        - mcpe
+        - mctest
+        - realms
+        - web
+      name: Awaiting Response
+      shortcut: ar
+      message: |-
+        %awaiting_response%
+
         %quick_links%
       fillname: []
```

Comment:
I'm not entirely sure how this is intended to be used, but maybe it would make sense to add a variable to this one for the question?